diff --git a/node_modules/react-native-svg/android/build.gradle b/node_modules/react-native-svg/android/build.gradle
index 4fe912c..a3cba29 100644
--- a/node_modules/react-native-svg/android/build.gradle
+++ b/node_modules/react-native-svg/android/build.gradle
@@ -85,7 +85,7 @@ def getFrescoVersion() {
 def FRESCO_VERSION = getFrescoVersion()
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    compileSdk safeExtGet('compileSdkVersion', 36)
     namespace "com.horcrux.svg"
     def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
     if (agpVersion.tokenize('.')[0].toInteger() >= 8) {
